doc/release-notes.md

```diff
@@ -64,61 +64,215 @@ platform.
 Notable changes
 ===============
 
+New user documentation
+----------------------
+
+- [Reduce memory](https://github.com/bitcoin/bitcoin/blob/master/doc/reduce-memory.md)
+  suggests configuration tweeks for running Bitcoin Core on systems with
+  limited memory. (#16339)
+
 New RPCs
 --------
 
 - `getbalances` returns an object with all balances (`mine`,
   `untrusted_pending` and `immature`). Please refer to the RPC help of
   `getbalances` for details. The new RPC is intended to replace
-  `getunconfirmedbalance` and the balance fields in `getwalletinfo`, as well as
-  `getbalance`. The old calls may be removed in a future version.
+  `getbalance`, `getunconfirmedbalance`, and the balance fields in
```

Comment:
Is second comma redundant?

Reply:
Among writers, whether or not to always use the serial comma is our version of vi versus emacs.  [Here's](https://www.quickanddirtytips.com/education/grammar/serial-comma) the first search result for the issue, which seems reasonably descriptive and even-handed.  I fall into the camp of people who always use the serial comma, but I willing to do lazy commas if necessary with only a few sotto voce grumblings as if you had also asked me to work in emacs.  :-)